pep-0501.txt

```diff
@@ -413,65 +547,83 @@ printf-style formatting. The runtime parsing and interpolation overhead for
 logging messages also poses a problem for extensive logging of runtime events
 for monitoring purposes.
 
-While beyond the scope of this initial PEP, interpolation template support
+While beyond the scope of this initial PEP, template literal support
 could potentially be added to the logging module's event reporting APIs,
 permitting relevant details to be captured using forms like::
 
-    logging.debug(i"Event: {event}; Details: {data}")
-    logging.critical(i"Error: {error}; Details: {data}")
+    logging.debug(t"Event: {event}; Details: {data}")
+    logging.critical(t"Error: {error}; Details: {data}")
 
 Rather than the current mod-formatting style::
 
     logging.debug("Event: %s; Details: %s", event, data)
     logging.critical("Error: %s; Details: %s", event, data)
 
-As the interpolation template is passed in as an ordinary argument, other
+As the template literal is passed in as an ordinary argument, other
 keyword arguments would also remain available::
 
-    logging.critical(i"Error: {error}; Details: {data}", exc_info=True)
+    logging.critical(t"Error: {error}; Details: {data}", exc_info=True)
 
 As part of any such integration, a recommended approach would need to be
 defined for "lazy evaluation" of interpolated fields, as the ``logging``
 module's existing delayed interpolation support provides access to
 `various attributes <https://docs.python.org/3/library/logging.html#logrecord-attributes>`__ of the event ``LogRecord`` instance.
 
-For example, since interpolation expressions are arbitrary Python expressions,
+For example, since template literal expressions are arbitrary Python expressions,
 string literals could be used to indicate cases where evaluation itself is
 being deferred, not just rendering::
 
-    logging.debug(i"Logger: {'record.name'}; Event: {event}; Details: {data}")
+    logging.debug(t"Logger: {'record.name'}; Event: {event}; Details: {data}")
 
 This could be further extended with idioms like using inline tuples to indicate
 deferred function calls to be made only if the log message is actually
 going to be rendered at current logging levels::
 
-    logging.debug(i"Event: {event}; Details: {expensive_call, raw_data}")
+    logging.debug(t"Event: {event}; Details: {expensive_call, raw_data}")
 
 This kind of approach would be possible as having access to the actual *text*
 of the field expression would allow the logging renderer to distinguish
 between inline tuples that appear in the field expression itself, and tuples
 that happen to be passed in as data values in a normal field.
 
 
+Comparison to PEP 675
+=====================
+This PEP is similar in its goals as :pep:`675`.
```

Comment:
Missing empty line after the heading

Reply:
Fixed in [my suggestion](https://github.com/python/peps/pull/3047#discussion_r1133055639) BTW, along with other fixes